include/openenclave/corelibc/wchar.h

```diff
@@ -1,2 +1,11 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
+
+#ifndef _OE_WCHAR_H
+#define _OE_WCHAR_H
+
+#include <openenclave/bits/types.h>
```

Comment:
Does it make sense to add a disclaimer that these headers are intended only for internal use?

Reply:
Ya good point. At least until we can formalize what we want to support